pylint/checkers/classes/class_checker.py

```diff
@@ -588,6 +588,11 @@ def _has_same_layout_slots(slots, assigned_value):
         "subclassed-final-class",
         "Used when a class decorated with typing.final has been subclassed.",
     ),
+    "W0241": (
```

Comment:
```suggestion
    "W0244": (
```

I just noticed that there is `E0241`. I know we don't really enforce this right now, but I think we're going to move to making the last two digits exclusive within a checker class.